src/main/java/fi/aalto/cs/apluscourses/model/Group.java

```diff
@@ -0,0 +1,76 @@
+package fi.aalto.cs.apluscourses.model;
+
+import fi.aalto.cs.apluscourses.intellij.services.PluginSettings;
+import fi.aalto.cs.apluscourses.utils.CoursesClient;
+import java.io.IOException;
+import java.io.InputStream;
+import java.net.URL;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import org.jetbrains.annotations.NotNull;
+import org.json.JSONArray;
+import org.json.JSONObject;
+import org.json.JSONTokener;
+
+public class Group {
+
+  private long id;
+
+  private List<String> memberNames;
```

Comment:
xm... do you know if the names (of the `full_name` uniqueness is guaranteed on the A+ side)?

to ensure we're safe I suggest to include the `id` of those students that form a pair. eventually, it would make sense to use a bit more complex object to represent a student (maybe a class `Student` even), but atm I'd at least encourage to include the identifiers. made a `group` between our test users on the dev O1 in: 
<details>
  <summary><a href="https://plus.cs.aalto.fi/api/v2/courses/163/mygroups/">/api/v2/courses/163/mygroups/</a></summary>

```json
{
    "count": 1,
    "next": null,
    "previous": null,
    "results": [
        {
            "id": 1284,
            "url": "https://plus.cs.aalto.fi/api/v2/courses/163/mygroups/1284/",
            "members": [
                {
                    "id": 19445,
                    "url": "https://plus.cs.aalto.fi/api/v2/users/19457/",
                    "full_name": "Kuusikymmentäneljä TESTI-Opiskelija"
                },
                {
                    "id": 19446,
                    "url": "https://plus.cs.aalto.fi/api/v2/users/19458/",
                    "full_name": "Kuusikymmentäviisi TESTI-Opiskelija"
                }
            ],
            "timestamp": "2020-07-02T09:51:12.035572+03:00"
        }
    ]
}
```
</details>

Reply:
Good point! Willl do